app/recordtransfer/views/post_submission.py

```diff
@@ -153,40 +139,44 @@ def get_form_kwargs(self) -> dict[str, Any]:
 
     def form_valid(self, form: BaseModelForm) -> HttpResponse:
         """Handle valid form submission."""
-        response = super().form_valid(form)
-        messages.success(self.request, self.success_message)
-        return response
+        super().form_valid(form)
+        response = render(
+            self.request,
+            self.template_name,
+            self.get_context_data(),
+        )
+        return trigger_client_event(
+            response,
+            "showSuccess",
+            {
+                "value": self.update_success_message,
```

Comment:
This message is only used once in the class, I think it's unnecessary to have it included as a class member. This function is a bit easier to read without having to go looking for `self.update_success_message`

```suggestion
                "value": gettext("Group updated"),
```

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/608/commits/59230f075e389318c60bf429c01d791e59b18177